src/error.ts

```diff
@@ -557,6 +586,34 @@ export class MongoAzureError extends MongoRuntimeError {
   }
 }
 
+/**
+ * A error generated when the user attempts to authenticate
+ * via GCP, but fails.
+ *
+ * @public
+ * @category Error
+ */
+export class MongoGCPError extends MongoRuntimeError {
```

Comment:
Should this be

```suggestion
export class MongoGCPError extends MongoOIDCError {
```

?

Reply:
Updated this as well as `MongoAzureError`.